src/main/resources/application-nomis.yml

```diff
@@ -3,7 +3,7 @@ spring:
   jpa:
     properties:
       hibernate:
-        dialect: org.hibernate.community.dialect.Oracle10gDialect
+        dialect: org.hibernate.community.dialect.OracleLegacyDialect
```

Comment:
switch to new legacy dialect that queries the database to find out the version to use